docs/05-GETTING_STARTED.md

```diff
@@ -0,0 +1,43 @@
+##Overview
+
+There are 4 important files needed to make restacker work.
+
+###Restacker.yml
+
+First, you need to configure your `~/.restacker/restacker.yml` file.
+
+This file is what allows your command line to connect and access the correct account.
+
+Validation:
+Keep tweaking this file until you can run `restacker list` and get a list of all the stacks.
+
+###Template.json
+
+Once this is complete, you need to get your hands on a template JSON file.
+
+###Parameters.yml
+
+After you have this you need to fill out your parameters.yml file. If your template has explanations for each field be sure to read them. 
```

Comment:
Remove "After you have this" - a bit superfluous.
Remove "If your template has explanations for each field be sure to read them" - not sure what you mean here.
You can reference the Parameter sample file that will reside in `infrastructure/parameters/` (currently in a pending PR)